vital/types/camera_intrinsics.h

```diff
@@ -145,13 +151,26 @@ class VITAL_EXPORT simple_camera_intrinsics :
                            const vector_2d& principal_point,
                            const double aspect_ratio=1.0,
                            const double skew=0.0,
+                           const unsigned int image_width=0,
+                           const unsigned int image_height=0,
                            const vector_t dist_coeffs=vector_t())
   : focal_length_(focal_length),
     principal_point_(principal_point),
     aspect_ratio_(aspect_ratio),
     skew_(skew),
+    image_width_(image_width),
+    image_height_(image_height),
     dist_coeffs_(dist_coeffs)
-  {}
+  {
+    if (image_width_ == 0)
+    {
+      image_width_ = static_cast<unsigned int>(std::ceil(principal_point(0)*2));
```

Comment:
While this is usually the best way to guess the image width and height, I'm wondering if we should guess at all.  Maybe we should just leave the width and height at 0 to indicate that it is unknown?  As it is now, there is no way of knowing if this is the true dimensions or just a guess.  I'm open for discussion on this.

Reply:
Using 0 as the default value, meaning that they are unknown, seems better. I will do the changes for the rpc camera too.